### PR TITLE
Add iml-common dependency

### PR DESCRIPTION
**Pull request by tanabarr**
add directory with most recent release from pypi to provide the iml-common python package on copr

### CODE REVIEW

**Comment by brianjmurrell** [comment]
I'm not sure if or why something would need to be fixed on Copr's end.  I have built a number of (properly constructed) PyPi modules in our Copr repo.

This is more likely something our module is doing wrong and Copr is showing us.

But that said, I'm afraid the error message in the `builder-live.log` isn't telling me much though.

Probably as a first step, the module needs to be built locally where the environment can be debugged.

File: python-iml-common/python-iml-common.spec [hunk 1]
```diff
@@ -12,7 +12,6 @@ Source0:        https://github.com/intel-hpdd/iml-common/archive/v1.0.3.tar.gz
 BuildArch:      noarch
  
 BuildRequires:  python2-devel
-BuildRequires:  python-setuptools-scm
```
Comment (brianjmurrell) [inline python-iml-common/python-iml-common.spec]: If you created this specfile with `pyp2rpm` and that tool put this _BuildRequires_ into the specfile then the Python module metadata is specifying this as a build dependency.  That would be where this needs to be fixed.

Comment (tanabarr) [inline python-iml-common/python-iml-common.spec]: yes was removed before you commented

**Comment by tanabarr** [comment]
as @brianjmurrell has pointed out, this should be enabled directly on copr if possible and build directly from Pypi. This was failing due to the incompatibility of using setuptools_scm with pyp2rpm which generates an invalid dependency in the spec file from the package metadata. 
https://github.com/intel-hpdd/iml-common/pull/5 replaces the use of setuptools_scm for deriving version from git tag with the routine in pre-commit hook.

**Pull request closed without merging**